README.md

```diff
@@ -45,6 +45,13 @@ Use the following plugins and guides to integrate Bugsnag with various framework
 | Express | [@bugsnag/plugin-express](packages/plugin-express)  | [Express docs](https://docs.bugsnag.com/platforms/javascript/express) |
 | Restify | [@bugsnag/plugin-restify](packages/plugin-restify)  | [Restify docs](https://docs.bugsnag.com/platforms/javascript/restify) |
 
+### Mobile
+
+| Framework  | Bugsnag plugin | Documentation |
```

Comment:
Toyed with this (see #525), but the `@bugsnag/expo` is a standalone notifier, not a plugin, so it didn't make sense. We could just have a section of copy here though under the title "Mobile".

Reply:
Seems clear with s/Bugsnag plugin/Bugsnag notifier/ 🤔